.github/workflows/test-loaders.yml

```diff
@@ -0,0 +1,17 @@
+name: Daily Loaders Test
+on:
```

Comment:
Let’s also run on `.github/workflows/test-loaders.yml` or related TS files changes.

Debugging cron-based workflows are hard (because you need to wait a day until they will run). Adding `pull_request` and `push` with `paths` (like [here](https://github.com/hplush/slowreader/blob/main/.github/workflows/docs.yml)) can simplify testing changes.

Reply:
Maybe we should also run tests in commits/PRs chinging `core/loader/*.ts`? What do you think?

Reply:
I agree, additional test for updated loaders on real data is a great idea. If developer will forget to run test locally it is run by the way automatically. I will add paths for push and РR.